setup.py

```diff
@@ -4,7 +4,7 @@
     long_description = f.read()
 
 setup(name='bond-home',
-      version='0.0.7',
+      version='0.1.0',
```

Comment:
Please revert this change

Reply:
Will do.  I can't remember why I even changed this.